apicast/src/configuration_loader.lua

```diff
@@ -105,6 +107,12 @@ function boot.init(configuration)
     ngx.log(ngx.EMERG, 'cache is off, cannot store configuration, exiting')
     os.exit(0)
   end
+
+  local keycloak_config = _M.init(nil, "keycloak")
```

Comment:
This is pretty poor interface and should be improved. Passing nil is just weird.